2025-08-14 09:34:43,246 - ERROR - Необработанное исключение: ValueError: invalid literal for int() with base 10: ''
2025-08-14 09:34:43,247 - ERROR - Трассировка стека:
Traceback (most recent call last):
  File "c:\Users\Alex\Competitions_TT_v1.2\main.py", line 3655, in tool_page
    page()
    ~~~~^^
  File "c:\Users\Alex\Competitions_TT_v1.2\main.py", line 3826, in page
    number_games = int(txt[:t])
ValueError: invalid literal for int() with base 10: ''
2025-08-14 09:34:46,874 - ERROR - Необработанное исключение: ValueError: invalid literal for int() with base 10: ''
2025-08-14 09:34:46,875 - ERROR - Трассировка стека:
Traceback (most recent call last):
  File "c:\Users\Alex\Competitions_TT_v1.2\main.py", line 3655, in tool_page
    page()
    ~~~~^^
  File "c:\Users\Alex\Competitions_TT_v1.2\main.py", line 3826, in page
    number_games = int(txt[:t])
ValueError: invalid literal for int() with base 10: ''
